package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -42,6 +42,12 @@ def to_json(self) -> str:
         """Returns session_store information in json format after converting PosixPath to string"""
         session_dict = {}
         for key, value in self.data.items():
+            if key == "git":
+                import git  # pylint: disable=import-outside-toplevel
```

Comment:
Let's `try...catch` this. The original concern was not about being outside of a git directory, but more like not having git installed in your system.

Reply:
sure, also if git is not available then it just won't return have git in the session_store dictionary.